src/commands/account/export_account/using_seed_phrase/mod.rs

```diff
@@ -0,0 +1,96 @@
+use color_eyre::eyre::WrapErr;
+
+#[derive(Debug, Clone, interactive_clap::InteractiveClap)]
+#[interactive_clap(input_context = super::ExportAccountContext)]
+#[interactive_clap(output_context = ExportAccountFromSeedPhraseContext)]
+pub struct ExportAccountFromSeedPhrase {
+    #[interactive_clap(named_arg)]
+    /// Select network
+    network_config: crate::network::Network,
+}
+
+#[derive(Clone)]
+pub struct ExportAccountFromSeedPhraseContext(crate::network::NetworkContext);
+
+impl ExportAccountFromSeedPhraseContext {
+    pub fn from_previous_context(
+        previous_context: super::ExportAccountContext,
+        _scope: &<ExportAccountFromSeedPhrase as interactive_clap::ToInteractiveClapContextScope>::InteractiveClapContextScope,
+    ) -> color_eyre::eyre::Result<Self> {
+        let config = previous_context.global_context.config.clone();
+        let account_id = previous_context.account_id.clone();
+
+        let on_after_getting_network_callback: crate::network::OnAfterGettingNetworkCallback =
+            std::sync::Arc::new({
+                move |network_config| {
+                    #[cfg(target_os = "macos")]
+                    {
+                        if let Ok(password_list) =
+                            super::using_web_wallet::get_password_list_from_macos_keychain(
```

Comment:
If you need this function in all three submodules, move it to the parent instead of keeping it in the `using_web_wallet` module.

Apply this reasoning to other similar functions.

Reply:
Done